ci/tasks/pip-audit.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+pip install -r requirements-dev.txt
```

Comment:
For the audit you can install just `pip-audit` to speed this up a bit

Reply:
Done. Still works and it's definitely lighter.